pgxgeom_test.go

```diff
@@ -115,6 +115,23 @@ func TestCodecScanValue(t *testing.T) {
 	})
 }
 
+func TestCodecScanValueGeography(t *testing.T) {
+	defaultConnTestRunner.RunTest(context.Background(), t, func(ctx context.Context, tb testing.TB, conn *pgx.Conn) {
+		tb.Helper()
+		for _, format := range []int16{
+			pgx.BinaryFormatCode,
+			pgx.TextFormatCode,
+		} {
+			tb.(*testing.T).Run(strconv.Itoa(int(format)), func(t *testing.T) {
+				var geom geom.T
+				err := conn.QueryRow(ctx, "select ST_SetSRID('POINT(1 2)'::geography, 4326)", pgx.QueryResultFormats{format}).Scan(&geom)
+				assert.NoError(t, err)
+				// assert.Equal(t, mustNewGeomFromWKT(t, "POINT(1 2)", 4326).SetSRID(4326).ToEWKBWithSRID(), geom.ToEWKBWithSRID())
```

Comment:
This test can be updated to match 0043bb365a1e40322ec20c0ef6d1d551f9ec097e.

Reply:
Done.